interface/main/left_nav.php

```diff
@@ -1534,6 +1549,35 @@ function selpopup(selobj) {
 
 <script language='JavaScript'>
 syncRadios();
+
+$('#showhide').click( function() {
+	var fa=$('#fa-arrows');
+	var exp = fa.hasClass('fa-angle-double-right');
+	var m = parent.document.getElementById("fsbody");
+	var r = parent.document.getElementById("fsright");
+	var nav = parent.document.getElementsByName("left_nav")[0];
+	var srv = <?php echo json_encode(array(
+	    'ahide' => $ahide,
+	    'whide' => ($_SESSION['language_direction'] == 'ltr' ? '8,*' : '*,8'),
+	    'ashow' => $ashow,
+	    'wshow' => ($_SESSION['language_direction'] == 'ltr' ? $GLOBALS['gbl_nav_area_width'].',*' : '*,'.$GLOBALS['gbl_nav_area_width']),
+	)); ?>;
+	if (m.cols == srv.whide) {
+ 	  m.frameborder = '1';
+ 	  r.frameborder = '1';
+		m.cols = srv.wshow;
+  	$('#fa-arrows').removeClass(srv.ashow);
+  	$('#fa-arrows').addClass(srv.ahide);
+  	nav.scrolling = 'yes';
+	} else {
+	  m.frameborder = '0';
+	  r.frameborder = '0';
+		m.cols = srv.whide;
+  	$('#fa-arrows').removeClass(srv.ahide);
+  	$('#fa-arrows').addClass(srv.ashow);
+  	nav.scrolling = 'no';
+	}
+});
```

Comment:
The question will become does it merit removing the Show/Hide menu functionality from the title bar? Or will users be annoyed whom use the Show/Hide links be annoyed by the crack that exists when closing with this new arrow method?